CleverPush/CleverPush/Source/CPUtils.m

```diff
@@ -227,4 +226,14 @@ + (BOOL)isEmpty:(id)thing
         && [(NSArray *)thing count] == 0);
 }
 
+#pragma mark -  Openup given URL in a SFSafariViewController.
++ (void)openSafari:(NSURL*)URL {
+    if (URL) {
+        if ([SFSafariViewController class] != nil) {
```

Comment:
What's the purpose of this check exactly? Is it not available on some iOS versions?

Reply:
@gbhrdt SFSafariViewController was introduced at iOS 9, so for the before versions I have made this check.